src/webui-embed.html

```diff
@@ -124,6 +150,12 @@
                         el.innerHTML = (stats[e]*100) + "%";
                     else if (e == "allow_self_select")
                         el.innerHTML = stats[e] == 1 ? "Yes" : "No";
+                    else if (e == "round_hashes")
+                    {
+                        el.innerHTML = (stats[e]*100/stats["network_difficulty"]).toFixed(2) + "%"
```

Comment:
Why is total accumulated pool diff / *current* network diff interesting? Surely you want to compare pool rounds accumulated diff / accumulated network diff of round?

Reply:
diff (it's called job.target in src/pool.c) in miner task description is an estimate of number of hashes to be done by miner to solve task on average.
I will use term hashes instead of diff to describe done work by pool workers.


pool_stats.round_hashes estimate number of hashes done by pool workers since the last found block and it will be set to zero (0) after every found block.

pool_stats.round_hashes != total accumulated pool diff
pool_stats.round_hashes == accumulated shares diff submitted by workers since the last found block

Reply:
My point is you're comparing pool accumulated round diff to only the current network block diff. Not the network diff covering the same period. I'm just trying to wrap my head round why you feel this (as you have it) is an important metric.

Reply:
It is expected to have a block as soon as 100% reached on average.